services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/create_unit/assign_a_new_activity.tsx

```diff
@@ -91,16 +115,54 @@ const AssignANewActivity = ({ numberOfActivitiesAssigned, showDiagnosticBanner }
     </section>)
   }
 
+  const suggestedActivitiesList = activitiesToSuggest.length > 0 &&
+  (
+    <div>
+      <h1 className="evidence-header">Browse the newest Reading for Evidence activities</h1>
+      <table className="data-table suggested-activities-table">
+        <tr className="data-table-headers">
+          <th className="data-table-header">Tool</th>
+          <th className="data-table-header activity-header">Activity</th>
+          <th className="data-table-header topics-header">Topics</th>
+          <th className="data-table-header">Date released</th>
+        </tr>
+        <tbody className="data-table-body">
+          <div className="list-item">
+            {activitiesToSuggest.map((a) => {
+              return (
+                <tr className="data-table-row" key={a.id}>
+                  <span className="tool-col"><img alt={evidenceToolIcon.alt} src={evidenceToolIcon.src} /></span>
+                  <td className="data-table-row-section name-col">
+                    <a href={`/assign/activity-library?activityClassificationFilters[]=evidence&search=${encodeURI(a.name)}`}>{a.name}</a>
+                  </td>
+                  <td className="data-table-row-section topics-col">
+                    {a.topics.map((t, i) => {
+                      return <p key={i}>{t.join(" / ")}</p>
+                    })}
+                  </td>
+                  <td className="date-col">{a.publication_date}</td>
+                  <td className="preview-col"><a href={`/activity_sessions/anonymous?activity_id=${a.id}`} rel="noopener noreferrer" target="_blank">Preview</a></td>
+                  <td className="select-col"><button className="quill-button secondary medium focus-on-light outlined select-suggested" onClick={() => window.location.href = `/assign/activity-library?activityClassificationFilters[]=evidence&search=${encodeURI(a.name)}`} type="button">Select</button></td>
+                </tr>
+              )
+            })}
+          </div>
+        </tbody>
+      </table>
+    </div>
+  )
+
   return (
-    <div className="assign-a-new-activity-container white-background-accommodate-footer">
+    <div className="assign-a-new-activity-container">
       <div className="assign-a-new-activity container">
         <h1>Find the perfect writing activities for your students.</h1>
-        {diagnosticBanner}
-        <div className="minis">{minis(diagnosticBannerShowing)}</div>
         <p className="previously-assigned-activities">
-        You have {numberOfActivitiesAssigned} {numberOfActivitiesAssigned === 1 ? 'activity' : 'activities'} assigned.&nbsp;
-          <a href="/teachers/classrooms/activity_planner">View assigned activities</a>
+          You have {numberOfActivitiesAssigned} {numberOfActivitiesAssigned === 1 ? 'activity' : 'activities'} assigned.&nbsp;
+          <button className="view-assigned-activities" type="button"><a href="/teachers/classrooms/activity_planner">View assigned activities</a></button>
```

Comment:
Can we just give the anchor tag that className? It feels weird to have an anchor nested inside of a button.